src/components/Insights/deprecated/InsightList/insightCards/NPlusOneInsight/index.tsx

```diff
@@ -46,8 +45,8 @@ export const NPlusOneInsight = (props: NPlusOneInsightProps) => {
       props.onJiraTicketCreate(props.insight, undefined, event);
   };
 
-  const spanName = props.insight.clientSpanName || undefined;
-  const spanCodeObjectId = props.insight.clientSpanCodeObjectId || undefined;
+  // const spanName = props.insight.clientSpanName || undefined;
```

Comment:
Pls add comment when it should be removed

Reply:
Added the comments to `@deprecated` annotations